docs/user/about/index.rst

```diff
@@ -68,7 +53,23 @@ we've brought most of the most important ones below.
 ⏩ :doc:`/support`
    Read this before asking for help: How to get support and where.
 
+⏩ :doc:`/glossary`
+   A useful index of terms used in our docs
```

Comment:
This seems like a wrong location for our Glossary?

Is it because you think the discoverability of the page itself isn't valuable? I tend to agree with that. So kind of :broom: 'ing into this corner seems fine until it has so many terms that it's a real resource.

Reply:
I think we need some other reference index to hide this stuff under, rather than in the top-level nav, is the real answer.